vita3k/gui/src/app_context_menu.cpp

```diff
@@ -559,8 +559,12 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
             ImGui::SetCursorPosX((WINDOW_SIZE.x / 2.f) + (20.f * SCALE.x));
         }
         if (ImGui::Button(common["ok"].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(emuenv.cfg.keyboard_button_cross)) {
-            if (context_dialog == lang["app_delete"])
+            if (context_dialog == lang["app_delete"]) {
                 delete_app(gui, emuenv, app_path);
+                fs::remove_all(ADDCONT_PATH);
+                fs::remove_all(LICENSE_PATH);
+                fs::remove_all(SAVE_DATA_PATH);
+            }
```

Comment:
revert

Reply:
if delete applation, only remove app path, not delete save data and dlc, does not match the description

Reply:
I know what i said no worry, revert it 👌